client/luascripts/hf_bruteforce.lua

```diff
@@ -60,46 +66,51 @@ local function help()
     print(usage)
 end
 ---
--- Exit message
-local function exitMsg(msg)
+--- Print user message
+local function msg(msg)
  print( string.rep('--',20) )
+ print('')
  print(msg)
+ print('')
  print( string.rep('--',20) )
- print()
 end
 ---
 -- Start
 local function main(args)
 
-    print( string.rep('--',20) )
-    print( string.rep('--',20) )
-    print()
     local timeout = 0
     local start_id = 0
-    local end_id = 0xFFFFFFFF
+    local end_id = 0xFFFFFFFFFFFFFF
+    local mftype = ''
 
-    for o, a in getopt.getopt(args, 'e:s:t:h') do
+    for o, a in getopt.getopt(args, 'e:s:t:h:T:') do
         if o == 's' then start_id = a end
         if o == 'e' then end_id = a end
         if o == 't' then timeout = a end
+        if o == 'T' then mftype = a end
```

Comment:
This idea of  two params that is the same but differ in capital or not,  is not recommend.
-t  vs -T  is confusing.
We need to come up with a better param name to make it clear and easy to understand.

I also suggest having **mfc** as default type,   making the new param optional from original expected behavior.

Reply:
Cool will update the script tonight and do a new pull request 👍

Reply:
I changed the option to `x` for Mifare type and set the default to __mfc__.  Have also added a new line for running when attached via USB and kept original command to run via BlueShark addon.

Reply:
Actually just noted that I didn't `git add` my final change to add the old BlueShark addon command back. Happy to redo that. Sorry for hassle